quesma/elasticsearch/mappings.go

```diff
@@ -0,0 +1,62 @@
+// Copyright Quesma, licensed under the Elastic License 2.0.
+// SPDX-License-Identifier: Elastic-2.0
+package elasticsearch
+
+import (
+	"github.com/rs/zerolog/log"
+	"maps"
+	"quesma/elasticsearch/elasticsearch_field_types"
+	"quesma/schema"
+)
+
+func ParseMappings(namespace string, mappings map[string]interface{}) map[string]schema.Column {
+	result := make(map[string]schema.Column)
+
+	properties := mappings["properties"].(map[string]interface{})
```

Comment:
I think it would be good to add sanity check to check whether "properties" key exists